File: src/de/geosearchef/rtsIO/js/UnitData.java
package de.geosearchef.rtsIO.js;

import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@NoArgsConstructor
public class UnitData {
    private String name;
    private String[] files;
    private float maxHp;
    private float movementSpeed;
    private int projectile;
    private int attackCooldown;
    private float attackRange;
}

File: public/play/game/unitData.js
var unitData = [

    {"name": "Scout", "files": ["scoutBody.svg", "scoutTurret.svg"], "maxHp": 100, "movementSpeed": 2.0, "projectile": 0, "attackCooldown": 1500, "attackRange": 7},
    {"name": "Tank", "files": ["tankBody.svg", "tankTurret.svg"], "maxHp": 200, "movementSpeed": 1.2, "projectile": -1},
    {"name": "Artillery", "files": ["artyBody.svg", "artyTurret.svg"], "maxHp": 120, "movementSpeed": 0.8, "projectile": -1},

];

function getUnitSize(typeID) {
    return 1.0;
}
